day-1/inventories/staging/group_vars/all/vars.yaml

```diff
@@ -15,7 +15,7 @@ hoprd:
       promiscuous:
         enabled: true
         max_channels: 10
-        network_quality_threshold: 0.6
+        network_quality_threshold: 0.5
```

Comment:
Equivalent to 5 consecutive missed pings, if the previous value was 1.0. but 1 missed ping, if the previous value was 0.6.